src/views/GovernanceOverview/index.scss

```diff
@@ -0,0 +1,41 @@
+.easi-governance-overview {
+  margin-bottom: 36px;
+
+  &__text {
+    font-size: 22px;
```

Comment:
🎨  Can we use `em` here instead of a fixed pixel size? Or maybe we can adjust the text with some inline utility classes?